### ISSUE TITLE
Oracle.EntityFrameworkCore spatial support

### ISSUE THREAD

**Issue by iulian0512** [oracle/dotnet-db-samples#89]
I can't find any information regarding support for spatial data, in the official efcore docs(https://docs.microsoft.com/en-us/ef/core/modeling/spatial) your provider is not listed and i could not find any info on oracle sites/forums. Thanks.

**Comment by alexkeh** [comment]
Oracle EF Core doesn't implement spatial data types currently.

**Comment by zhouzhoujinjin** [comment]
you can use devart to fix that,but it is not for free

**Comment by gscpw** [comment]
@alexkeh Ok great, it is not implemented. Would you consider reopening the issue for tracking this as a feature request?

**Comment by alexkeh** [comment]
@gscpw Is there a difference between your request and #190? That request is for UDT and NetTopologySuite support.

**Comment by gscpw** [comment]
@alexkeh I assumed #190 is only for UDT, while I considered #89 a request for NetTopologySuite. I suppose implementing NetTopologySuite should be way easier than adding a generic UDT support, so I think it's better to have different issues for each? In that case, NetTopologySuite implementation would not be blocked by whatever additional research generic UDT support requires.

**Comment by alexkeh** [comment]
@gscpw I considered #89 as a request for information, not a feature request. Back in 2020, there were no GitHub discussion forums, at least I don't remember them existing back then. People used the Issues areas for questions, bugs, and feature requests. (They actually still do out of habit.) For questions, I would close the issue once the question was answered.

That's what I did here. There are cases in which the OP clarified and said they wanted something else or additional. If that was the case, I would re-open the issue.

What I would recommend is if you are looking for NetTopologySuite specifically is to create a new issue that is unambiguous that it is a feature request. If other developers are looking for the same feature, they will upvote with a thumbs up emoji. That's an informal way the Oracle .NET team tracks developer interest in feature requests.

Both #89 and #190 have not received many votes compared to other Oracle EF Core and ODP.NET feature requests. That's a primary reason these features haven't been scheduled yet.

**Comment by alexkeh** [comment]
@iulian0512 @zhouzhoujinjin Is the Oracle EF Core NTS assembly working for you? Any problems you encountered?
